dev-docs/bidders/contentexchange.md

```diff
@@ -6,7 +6,7 @@ biddercode: contentexchange
 usp_supported: true
 schain_supported: true
 media_types: banner, video, native
-tcfeu_supported: false
+tcfeu_supported: true
```

Comment:
you need to provide a `gvl_id` header in order to claim tcfeu_supported - thanks